ruby-challenges.rb

```diff
@@ -19,17 +26,67 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+def sum_of array
+  array.sum
+end
 
+p sum_of nums_array1
+p sum_of nums_array2
 
 # --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a bike_info method that returns a sentence with all the data from the bike object.
 
 # Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
 
-
-
 # -------------------3b) Add the ability to pedal faster and brake. The pedal_faster method should increase the speed by a given amount. The brake method should decrease the speed by a given amount. The bike cannot go negative speeds.
 
 # Expected output example: my_bike.pedal_faster(10) => 10
 # Expected output example: my_bike.pedal_faster(18) => 28
 # Expected output example: my_bike.brake(5) => 23
 # Expected output example: my_bike.brake(25) => 0
+
+class Bike
+  
+  attr_accessor :model, :wheels, :current_speed
+
+  def initialize model
+
+    @model = model
+    @wheels = 2
+    @current_speed = 0
+
+  end
+
+  def bike_info
+
+    "The #{@model} bike has #{@wheels} wheels and is going #{@current_speed} mph."
+
+  end
+
+  def pedal_faster(go_faster)
+
+    @current_speed += go_faster
+
+  end
+
+  def brake(go_slower)
+
+    if @current_speed - go_slower <= 0
+       @current_speed = 0
+
+    else
+    
+      @current_speed -= slow_down
+
+    end
+
+  end
+
+end
+
+this_bike = Bike.new('Trek')
+
+p this_bike.bike_info
+p this_bike.pedal_faster(10)
+p this_bike.pedal_faster(18)
+p this_bike.brake(5)
+p this_bike.brake(25)
```

Comment:
Looks good! I'm not sure if it is just the GitHub UI, but I am seeing a lot of extra spaces around your code.